App.js

```diff
@@ -1,21 +1,39 @@
-import { StatusBar } from 'expo-status-bar';
 import React from 'react';
 import { StyleSheet, Text, View } from 'react-native';
+import { createAppContainer, createSwitchNavigator } from 'react-navigation';
+import AuthNavigator from './navigation/AuthNavigator';
+import HomeScreen from './screens/HomeScreen.js';
+// Firebase App (the core Firebase SDK) is always required and
+// must be listed before other Firebase SDKs
+import * as firebase from "firebase/app";
 
-export default function App() {
-  return (
-    <View style={styles.container}>
-      <Text>Open up App.js to start working on your app!</Text>
-      <StatusBar style="auto" />
-    </View>
-  );
-}
+// Add the Firebase services that you want to use
+import "firebase/auth";
+import "firebase/firestore";
 
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    backgroundColor: '#fff',
-    alignItems: 'center',
-    justifyContent: 'center',
-  },
-});
+// TODO: Replace the following with your app's Firebase project configuration
+const firebaseConfig = {
```

Comment:
Is having firebase api key out in the open like this is a good idea? It could be a security risk maybe?
What i'd recommend is:
1) creating a new config file.
2) Importing the config file into app.js.
3) Gitignore config file, so people can't see that file.
This way people won't have access to your api key.